lms/djangoapps/verify_student/views.py

```diff
@@ -1154,10 +1154,26 @@ def results_callback(request):
     except SoftwareSecurePhotoVerification.DoesNotExist:
         log.error("Software Secure posted back for receipt_id %s, but not found", receipt_id)
         return HttpResponseBadRequest("edX ID {} not found".format(receipt_id))
+    context = {
+        'id': attempt.user.id,
+        'email': attempt.user.email
+    }
+    email_template_context = {'platform_name': settings.PLATFORM_NAME}
     if result == "PASS":
         log.debug("Approving verification for %s", receipt_id)
         attempt.approve()
         status = "approved"
+        expiry_date = datetime.date.today() + datetime.timedelta(
+            days=settings.VERIFY_STUDENT["DAYS_GOOD_FOR"]
+        )
+        email_template_context['full_name'] = attempt.user.profile.name
+        email_template_context['expiry_date'] = expiry_date.strftime("%m/%d/%Y")
+        context['email_template_context'] = email_template_context
```

Comment:
Can we move above all into `send_verification_status_email()`, because looks like these values will be same for all status emails.